Disable network-bridge script by default to avoid
NetworkManager conflict

diff -up xen-3.2.0/tools/examples/xend-config.sxp.disable-bridge-by-default xen-3.2.0/tools/examples/xend-config.sxp
--- xen-3.2.0/tools/examples/xend-config.sxp.disable-bridge-by-default	2008-02-18 22:35:26.000000000 +0000
+++ xen-3.2.0/tools/examples/xend-config.sxp	2008-02-18 22:53:37.000000000 +0000
@@ -140,7 +140,15 @@
 # two fake interfaces per guest domain.  To do things like this, write
 # yourself a wrapper script, and call network-bridge from it, as appropriate.
 #
-(network-script network-bridge)
+#####################################
+#
+# We assume NetworkManager is enabled by default & managing public
+# interfaces, including their bridge membership. Xen's networking
+# setup thus serves no useful purpose.
+(network-script /bin/true)
+# Only uncomment this if you are using legacy network init scripts. This 
+# will *NOT* play nicely with NetworkManager
+#(network-script network-bridge)
 
 # The script used to control virtual interfaces.  This can be overridden on a
 # per-vif basis when creating a domain or a configuring a new vif.  The
@@ -160,12 +168,14 @@
 
 ## Use the following if network traffic is routed, as an alternative to the
 # settings for bridged networking given above.
+# NB: Obsolete. See note above for LAPTOP USERS
 #(network-script network-route)
 #(vif-script     vif-route)
 
 
 ## Use the following if network traffic is routed with NAT, as an alternative
 # to the settings for bridged networking given above.
+# NB: Obsolete. See note above for LAPTOP USERS
 #(network-script network-nat)
 #(vif-script     vif-nat)
 
